src/components/signup/SignupState.js

```diff
@@ -78,10 +78,11 @@ const Signup = ({ navigation, screenProps }: { navigation: any, screenProps: any
           await API.verifyUser({})
           const destinationPath = store.get('destinationPath')
           store.set('destinationPath')('')
```

Comment:
I would move this inside the if after navigating to destinationPath for clarity

Reply:
Indeed, good catch